### PR TITLE
ENH: add singularity CLI to wrap calls to the Singularity BIDS APP image

### PR DESCRIPTION
**Pull request by anilbey**

### CODE REVIEW

**Comment by anilbey** [comment]
@sebastientourbier we need to add some tests here I think

**Comment by sebastientourbier** [comment]
@all-contributors Add @anilbey as a contributor for code, test, ideas, and doc.

File: cmp/parser.py [hunk 1]
```diff
@@ -93,3 +93,22 @@ def get():
...
+    -------
+    p : object
+       Instance of :class:`argparse.ArgumentParser` class
+    """
+    p = get()
+    p.description = f"Entrypoint script of the BIDS-App Connectome Mapper version {__version__} via Singularity."
+    p.add_argument(
+        '--singularity_image',
+        type=str,
+        default=f'library://connectomicslab/default/connectomemapper-bidsapp:{__version__} ',
```
Comment (sebastientourbier) [inline cmp/parser.py:111]:
```suggestion
        default=f'library://connectomicslab/default/connectomemapper-bidsapp:{__version__}',
```

**Review by sebastientourbier — APPROVED** [review]

**Review by sebastientourbier — APPROVED** [review]
Seems ready to be merged in `release/v3.0.0`! Just removed the changes introduced in usage.rst doc page as this should be integrated by PR #112 and corrected the paths in the `.circleci/config.yml` file.

Thanks @anilbey for this contribution! 🚀

**Comment by anilbey** [comment]
Sure @sebastientourbier, I'm happy to help :)

**Pull request merged**